src/core/settings.c

```diff
@@ -1711,6 +1744,7 @@ QuicSettingsGetSettings(
     SETTING_COPY_FROM_INTERNAL(DatagramReceiveEnabled, Settings, InternalSettings);
     SETTING_COPY_FROM_INTERNAL(ServerResumptionLevel, Settings, InternalSettings);
     SETTING_COPY_FROM_INTERNAL(GreaseQuicBitEnabled, Settings, InternalSettings);
+    SETTING_COPY_FROM_INTERNAL(ReliableResetEnabled, Settings, InternalSettings);
```

Comment:
Remove this line. You already have the copy below.

Reply:
good catch! I can remove